source/query-bar.txt

```diff
@@ -56,6 +56,14 @@ options of the query.
 
      - The upper limit for the number of documents to return.
 
+.. note::
+
+   |compass| does not support the :manual:`$near
+   </reference/operator/query/near/>` and
+   :manual:`$nearSphere </reference/operator/query/nearSphere/>`
+   geospatial query operators in any of the query bar
+   fields.
```

Comment:
I *do* think you should include that second sentence explaining why...or just append it with a comma: 
> ...any of the query bar fields, because they do not...

Reply:
Thanks for the input, reading it now I think it's good to include.